frontend/src/components/MemberVotesList.test.tsx

```diff
@@ -90,6 +90,24 @@ describe("MemberVotesList", () => {
     assert.ok(within(filtered[0]).getByText("Max MUSTERMANN"));
   });
 
+  it("can search members by name with superfluous whitespace", async () => {
+    const { getAllByRole, getByRole } = render(
+      <MemberVotesList
+        memberVotes={memberVotes}
+        groups={[group]}
+        countries={[country]}
+      />,
+    );
+    const all = getAllByRole("listitem");
+    assert.strictEqual(all.length, 3);
+
+    await userEvent.type(getByRole("searchbox"), "max  muster");
+
+    const filtered = getAllByRole("listitem");
+    assert.strictEqual(filtered.length, 1);
+    assert.ok(within(filtered[0]).getByText("Max MUSTERMANN"));
```

Comment:
No need for a completely separate test case, but can you add an assertion below that tests that leading white space is ignored too?

Reply:
Done :)